tests_operations.py

```diff
@@ -61,4 +61,6 @@ def test_modulo():
     WHEN:  passed in to the function modulo
     THEN:  the resulting is the modulo of the two number
     """
-    pass
+    assert modulo(8, 2) == 0
+    assert modulo(9, 2) == 1
```

Comment:
We are missing the function definition. Please add that in otherwise it will break the build if we merge it in